internal/shared/messaging/proto/events.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
```

Comment:
Do you know why git didn't treat this as a `mv` while it did for most of the others?